packages/web-components/fast-components/src/card/index.ts

```diff
@@ -15,8 +27,80 @@ import { CardStyles as styles } from "./card.styles";
     name: "fast-card",
     template,
     styles,
+    shadowOptions: {
+        mode: "closed",
+    },
 })
-export class FASTCard extends Card {}
+export class FASTCard extends DesignSystemProvider
+    implements Pick<DesignSystem, "backgroundColor" | "neutralPalette"> {
+    /**
+     * Background color for the banner component. Sets context for the design system.
+     * @public
+     * @remarks
+     * HTML Attribute: background-color
+     */
+    @designSystemProperty({
+        attribute: "background-color",
+        default: "#FFFFFF",
+    })
+    public backgroundColor: string;
+
+    /**
+     * Background color for the banner component. Sets context for the design system.
+     * @public
+     * @remarks
+     * HTML Attribute: background-color
+     */
+    @attr({
+        attribute: "card-background-color",
+    })
+    public cardBackgroundColor: string;
+    private cardBackgroundColorChanged(prev: string | void, next: string | void): void {
+        if (next) {
+            const parsedColor = parseColorHexRGB(this.cardBackgroundColor);
+
+            if (parsedColor !== null) {
+                this.neutralPalette = createColorPalette(parsedColor);
```

Comment:
`createColorPalette` is a pure function, so I would recommend setting up a palette cache in this file so that we don't go through the work of creating duplicate palettes when the source color is the same.